sentinel-core/src/main/java/com/alibaba/csp/sentinel/Tracer.java

```diff
@@ -39,87 +35,81 @@
     protected Tracer() {}
 
     /**
-     * Trace provided {@link Throwable} and increment exception count to entry in current context.
+     * Trace provided {@link Throwable} to the resource entry in current context.
      *
      * @param e exception to record
      */
     public static void trace(Throwable e) {
-        trace(e, 1);
+        traceContext(e, ContextUtil.getContext());
     }
 
     /**
-     * Trace provided {@link Throwable} and add exception count to entry in current context.
+     * Trace provided {@link Throwable} to current entry in current context.
      *
      * @param e     exception to record
      * @param count exception count to add
      */
+    @Deprecated
     public static void trace(Throwable e, int count) {
         traceContext(e, count, ContextUtil.getContext());
     }
 
     /**
-     * Trace provided {@link Throwable} and add exception count to current entry in provided context.
+     * Trace provided {@link Throwable} to current entry of given entrance context.
      *
      * @param e     exception to record
-     * @param count exception count to add
-     * @since 1.4.2
+     * @param context target entrance context
+     * @since 1.8.0
      */
-    public static void traceContext(Throwable e, int count, Context context) {
+    public static void traceContext(Throwable e, Context context) {
         if (!shouldTrace(e)) {
             return;
         }
 
         if (context == null || context instanceof NullContext) {
             return;
         }
-
-        DefaultNode curNode = (DefaultNode)context.getCurNode();
-        traceExceptionToNode(e, count, context.getCurEntry(), curNode);
-    }
-
-    /**
-     * Trace provided {@link Throwable} and increment exception count to provided entry.
-     *
-     * @param e exception to record
-     * @since 1.4.2
-     */
-    public static void traceEntry(Throwable e, Entry entry) {
-        traceEntry(e, 1, entry);
+        traceEntryInternal(e, context.getCurEntry());
     }
 
     /**
-     * Trace provided {@link Throwable} and add exception count to provided entry.
+     * Trace provided {@link Throwable} and add exception count to current entry in provided context.
      *
      * @param e     exception to record
      * @param count exception count to add
      * @since 1.4.2
      */
-    public static void traceEntry(Throwable e, int count, Entry entry) {
+    @Deprecated
+    public static void traceContext(Throwable e, int count, Context context) {
         if (!shouldTrace(e)) {
             return;
         }
-        if (entry == null || entry.getCurNode() == null) {
+
+        if (context == null || context instanceof NullContext) {
             return;
         }
-
-        DefaultNode curNode = (DefaultNode)entry.getCurNode();
-        traceExceptionToNode(e, count, entry, curNode);
+        traceEntryInternal(e, context.getCurEntry());
     }
```

Comment:
The parameter `count` now is unused, do we have no scenes that need to use it? 
As `@Deprecated` is marked, maybe the caller `SentinelReactorSubscriber#L150` and `ReactorSphU#L60` use the newer method is better.

Reply:
As it's related to other adapter modules, it might be better to make changes in a new PR.